lib/ansible/modules/storage/hpe3par/ss_3par_cpg.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/python
+# Copyright: (c) 2018, Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+short_description: Manage HPE StoreServ 3PAR CPG
+author:
+  - Farhan Nomani (@farhan7500)
+  - Gautham P Hegde (@gautamphegde)
+description:
+  - Create and delete CPG on HPE 3PAR.
+module: ss_3par_cpg
+options:
+  cpg_name:
+    description:
+      - Name of the CPG.
+    required: true
+  disk_type:
+    choices:
+      - FC
+      - NL
+      - SSD
+    description:
+      - Specifies that physical disks must have the specified device type.
+  domain:
+    description:
+      - Specifies the name of the domain in which the object will reside.
+  growth_increment:
+    description:
+      - Specifies the growth increment the amount of logical disk storage
```

Comment:
Would be good to mention if this is measured in MiB (based on test code that seems correct?) same for the other growth_* options.